nvflare/fuel/utils/class_utils.py

```diff
@@ -137,7 +121,14 @@ def get_module_name(self, class_name) -> Optional[str]:
         Returns:
             The module name if found.
         """
-        return self._class_table.get(class_name, None)
+        if class_name not in self._class_table:
+            return None
+
+        modules = self._class_table.get(class_name, None)
+        if modules and len(modules) > 1:
+            raise ConfigError(f"There are multiple modules with the class_name:{class_name}, modules are: {modules}")
```

Comment:
Improve the text to:
f"Multiple modules have the class '{class_name}': {modules}. Please specify the class using its full path."

Reply:
updated.